tests/ruby/stats_spec.rb

```diff
@@ -29,6 +29,9 @@
         expect(results["total_query_time"].to_i).to be_within(200).of(750)
         expect(results["avg_query_time"].to_i).to be_within(50).of(250)
 
+        expect(results["total_xact_time"].to_i).to be_within(200).of(750)
```

Comment:
This is failing in CI:

```
  1) Stats SHOW STATS clients connect and make one query updates *_query_time and *_wait_time
     Failure/Error: expect(results["total_xact_time"].to_i).to be_within(200).of(750)
       expected 1203 to be within 200 of 750
     # ./stats_spec.rb:32:in `block (4 levels) in <top (required)>'
```

It passes locally for me inside the dev container, but I wonder if that shows that this approach is flawed.

Having said that, I do see other specs fail some of the time, like this, which I don't think I broke:

```
  1) Stats SHOW POOLS bad database name does not change any stats
     Failure/Error: expect(results["sv_idle"]).to eq("1")

       expected: "1"
            got: "2"

       (compared using ==)
     # ./stats_spec.rb:72:in `block (4 levels) in <top (required)>'
```